Bio/SeqUtils/ProtParam.py

```diff
@@ -75,9 +74,9 @@ class ProteinAnalysis:
     def __init__(self, prot_sequence, monoisotopic=False):
         """Initialize the class."""
         if prot_sequence.islower():
-            self.sequence = Seq(prot_sequence.upper(), IUPAC.protein)
+            self.sequence = Seq(prot_sequence.upper())
         else:
-            self.sequence = Seq(prot_sequence, IUPAC.protein)
```

Comment:
This is a pre-existing design choice, but the logic here puzzles me - what about mixed case? Wouldn't this be better:

```python
if prot_sequence.isupper():
    self.sequence = Seq(prot_sequence)
else:
    self.sequence = Seq(prot_sequence.upper())
```

Or far simpler, just:

```python
    self.sequence = Seq(prot_sequence.upper())
```

Reply:
It's a mystery to me too.

Reply:
We'll get to it once we fix `Seq` objects.